src/lib.rs

```diff
@@ -0,0 +1,80 @@
+use web_time::Instant;
+
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct ConversionResult {
+    pub value: String,
+    pub conversion_time: f32,
+    pub size_reduced_by: f32,
+}
+
+pub fn decode(zipson: String) -> anyhow::Result<ConversionResult> {
+    let start = Instant::now();
+    let zipson_value = serde_zipson::de::from_str::<serde_zipson::value::Value>(&zipson)?;
+    // counting only zipson parsing
+    let conversion_time = start.elapsed().as_micros() as f32 / 1000.;
```

Comment:
There is a [`Duration::as_millis`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_millis) method you could use instead of dividing by `1000` yourself. Without knowing where `ConversionResult` is used, I'd probably also either save the `Duration` directly or the nanoseconds in `u128` instead of converting to `f32`. And make the scale part of the variable name. I.e. instead of `conversion_time`, I'd name the field `conversion_time_in_ms` or something like that.

Reply:
fixed